tannerb9/MasksNowTN-New
# tb-News

# THIS PR IS NOT READY; JUST CREATED IT FOR SOME GITHUB ACTIVITY. I'LL MSG IN SLACK WHEN IT'S READY

# Description
Updated the original sample carousel to target correct data. Needs styling to correct wonky sizing.

## Type of change

# Testing Instructions
Run the app and ensure the carousel has three slides. Each slide contains a title and renders an image or text body if either exists.
